src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/AssemblyContigAlignmentSignatureClassifier.java

```diff
@@ -103,6 +104,12 @@ private static void decisionTreeForContigsWithTwoAlignments(final JavaRDD<Aligne
 
     // TODO: 10/27/17 add tests for these predicates
 
+    static boolean hasIncompletePicture(final AlignedContig contigWithOnlyOneConfigAnd2Aln) {
```

Comment:
Again see my other comment about renaming this to be more clear about what's going on, ie something like `doesNotSpanDuplicationBreakpoints` or something